File: Practice/src/coding_basic_training/ReverseString.py
def solution(my_string, s, e):
    arr = list(my_string)
    for i in range(s, s + int((e - s) / 2) + 1):
        temp = arr[i]
        arr[i] = arr[e - i + s]
        arr[e - i + s] = temp
        print(i, e - i + s, s, e)
    return "".join(arr)

File: Practice/src/coding_basic_training/ReverseString.js
function solution(my_string, s, e) {
    let arr = my_string.split("");
    for (let i = s; i <= s + (e - s) / 2; i++) {
        let temp = arr[i];
        arr[i] = arr[e - i + s];
        arr[e - i + s] = temp;
    }
    return arr.join("");
}
